### DISCUSSION TITLE
Official site and forums abandoned? No redirects or announcements.

### DISCUSSION THREAD

**Discussion by TheOldMan2000** [leokhoa/laragon#365] [General]
Hi,

I love learning to use Laragon and your forum community was an incredibly useful resource after switching from XXAMP on Win 10. I got so much help especially from users like @Pen-Y-Fan and it was such a huge wealth of knowledge.

I just can't understand why there is no announcement at the https://laragon.org/ site, the community links still point to the support forum at https://forum.laragon.org/ which has said the same message for weeks, perhaps months, "Working on it". There's no mention of it on the Twitter link either.

Laragon deserves better than this! Even a redirect for new users who haven't seen this Github would be helpful as I feel I can't recommend this great software anymore.

All the best for New Year and thank you again for Laragon.

**Comment by Pen-y-Fan** [comment]
Hi @TheOldMan2000 ,

Thanks for your kind words, I've used Lagaon for years. I wanted to share my knowledge with others. Laragon is free and until recently Leo hasn't asked for any donations. I used my time and expertise to help others. Laragon is the best dev environment on Windows!

Leo has stated the old forum, which was  NodeBB v2, had to be taken down for [security reasons](https://github.com/leokhoa/laragon/issues/348#issuecomment-1324050397).

I understand Leo is busy, and these discussions may or may not be a temporary solution. The closest to an announcement is [Welcome to Laragon Discussions!](https://github.com/leokhoa/laragon/discussions/344), however, this isn't clear if this discussion is a temporary solution. 

The link to 'community' from the Laragon.org site goes to [forum](https://forum.laragon.org/), but only displays a Working on it ... landing page. Rather than this discussion group.

Hi @leokhoa ,

I agree with TheOldMan2000, Laragon does deserve better, it does need some form of support. 

My questions:

- What are the long-term aims of the forum/discussions?
- Do you think the old NodeBB v2 forum can be upgraded? (given enough time)
- Can the link to the form be changed to this [discussion group](https://github.com/leokhoa/laragon/discussions), rather than an empty landing page?
- Can the 'how to ...' help pages be retrieved from the old forum and recreated as discussions?
- Or is it better to move to a more 'modern' option like [Discord](https://discord.com/)?

I would be happy to help if I can.

**Reply by TheOldMan2000** [reply]
Your welcome, thanks for the reply. Makes a little more sense knowing the reason, thanks. Upgrading from very old versions is rarely simple! Hopefully before attempting the upgrade, Leo remembered to take a backup. NodeBB tech support should be able to help too. 

I've owned forums for 2 decades now (for the most part self-hosted Invision Community from IPS) and TBH, I was quite shocked at the hosted price of NodeBB. I can only imagine that has a lot to do with things, migrating to a different platform is defintely a good idea, but just updating the old site links is a starter for now!

**Reply by codtechzofficial** [reply]
Hello @leokhoa . I will support your project by sponsoring the hosting of Laragon community forum. 

@Pen-y-Fan can you help me reaching out Leokhoa?

**Reply by Pen-y-Fan** [reply]
The majority of the broken links have been moved to the wiki. You can find the updated link to tutorials at #737. 

Hopefully, Laragon 7 will have the updated links to this Discussion forum and the Wiki. 

This discussion forum works well, so I wouldn't recommend moving to another one. 

Instead of moving, why not consider donating #383?

**Comment by charlesroper** [comment]
I've just found these discussion forums. I agree it would be good if the forum link on the official site pointed here, even if only temporarily. GitHub discussions are a pretty good forum, but the main benefit is that @leokhoa doesn't need to host or maintain or pay them himself. Also, most developers will already have a GitHub account, so there's no need to create yet another account on another forum.

Perhaps if we can get hold of the data from the old forum, we could all help contribute by adding them here or to the [wiki](https://github.com/leokhoa/laragon/wiki)?

**Reply by BiggerLimits** [reply]
### Thanks to the Binary Gods for my having found this refreshing news -- I'll sleep better tonight knowing Laragon lives on.

Laragon is, and the former Forum was,  too valuable to simply disappear, or left behind an awful graphic and cryptic "_Working on it_".  

It was like losing a trusted friend.   So much so that I had resorted to using the WayBack Machine to find old Forum information. 

**Many thanks** to Leo for his amazing gift and best wishes for many decades more of Laragon success.

**Comment by TheOldMan2000** [comment]
You can't beat a proper forum for tech support issues! So useful, a great searchable archive too.

**Reply by charlesroper** [reply]
GitHub Discussions _are_ a proper forum though, aren't they? Seems that way to me. Are there specific features you're missing here?

**Comment by TheOldMan2000** [comment]
It occurred to me to see what is available via the Internet Archive/Wayback Machine. Sadly not much. It was actually not that huge as forums go, according to what little is archived via the Wayback Machine, around 10000 topics in 6 categories, around 4500 posts according to the last archived page in October 2023 that I could find. 

I'm holding out that when Leo returns (his profile status says he is on vacation), he has a backup or can resolve the situation or provide access in some way for a conversion. It would be great to see it back again.

**Reply by BiggerLimits** [reply]
Ditto.

**Comment by Pen-y-Fan** [comment]
I've just noticed the link to **Community**, on the laragon.org site, now goes to a holding page. 

> ... I cannot keep the self-hosted forum and moving our discussions to GitHub

![Discussions](https://user-images.githubusercontent.com/40126936/215276418-87be19c3-0d00-4b6e-b437-2f2e834cb7b1.png "Forum replaced with Discussions")

I think this is the news we have been expecting, discussions can be used in place of the forum 🎉

**Comment by TheOldMan2000** [comment]
Thanks for the update, yes definitely. :)

**Comment by anthonycolucci** [comment]
I appreciate your good compliments; I've used Lagaon for a long time. I desired to impart my expertise to others. Laragon is free and until recently Leo hasn't asked for any donations[.](https://mybankinginformation.com/does-pizza-hut-take-ebt-cards/) I volunteered my time and skills to assist others. Laragon is the best dev environment on Windows!

**Comment by RadioMetaverse** [comment]
This is my first post.  I am struggling to learn to code but I am very persistent.  My problem is I forget things almost as fast as I learn them.  It makes it tough.  Being 72 years old does not help either as the mind is not as strong as it once was if it ever was.

I just want to say that it blows my mind that Le Ngoc Khoa has not been showered with donations to keep his self hosted site up and to profit substantially from Laragon.   I am on a fixed income but I am going to donate!  Thank you Le Ngoc Khoa for your fabulous work.

**Comment by kwsim539** [comment]
I agree with @Pen-y-Fan. Discord would be more suitable for a cpmmunity forum IMO.

**Reply by charlesroper** [reply]
Discord is generally not ideal for a support forum. It can't be indexed by search engines, and you have to have an account to access/view it. Getting a Discord account is not straightforward for everyone. Discord is great for private communities but not so great for support forums.

**Reply by kwsim539** [reply]
Got it. Thank you for pointing that out.
